### ISSUE TITLE
Suspend active Firewall after download to avoid time of downtime

### ISSUE THREAD

**Issue by Remi-SIB** [cdot65/pan-os-upgrade#140]
First at all, thanks for your great work !

**Is your feature request related to a problem? Please describe.**
when we have a HA pair of firewall, the active one is suspended just before downloading image, so it's in this state during all the download time. I think it could still be passive during the download in case something went wrong. It seems to occur when you donwload a base image and then download a second image.

**Describe the solution you'd like**
Suspend the active one only after all image are downloaded

**Describe alternatives you've considered**
Check the action of suspend active firewall and move it after all download action.

**Additional context**
![image](https://github.com/user-attachments/assets/b2679f4d-d780-4238-a94d-6aa2b35e9ef1)

**Comment by cdot65** [comment]
you're absolutely right, I changed the workflow for the web upgrade version but will revisit the CLI version too. thanks for bringing this up!
